aspnet/conceptual-overview/aspnet.rst

```diff
@@ -85,27 +101,50 @@ You can use any `OWIN <http://owin.org>`_-based middleware with ASP.NET Core. Se
 Servers
 -------
 
-The ASP.NET Core hosting model does not directly listen for requests, but instead relies on an HTTP :doc:`server </fundamentals/servers>` implementation to surface the request to the application as a set of feature interfaces that can be composed into an HttpContext. ASP.NET Core includes a managed cross-platform web server, called :ref:`Kestrel <kestrel>`, that you would typically run behind a production web server like `IIS <https://iis.net>`__ or `nginx <http://nginx.org>`__.
+The ASP.NET Core hosting model does not directly listen for requests; rather it relies on an HTTP :doc:`server </fundamentals/servers>` implementation to forward the request to the application. The forwarded request is wrapped as a set of feature interfaces that the application then composes into an ``HttpContext``.  ASP.NET Core includes a managed cross-platform web server, called :ref:`Kestrel <kestrel>`, that you would typically run behind a production web server like `IIS <https://iis.net>`__ or `nginx <http://nginx.org>`__.
+
+Content root
+------------
+
+The content root is the base path to any content used by the app, such as its views and web content. By default the content root is the same as application base path for the executable hosting the app; an alternative location can be specified with `WebHostBuilder`.
 
 Web root
 --------
 
-The Web root of your application is the root location in your project from which HTTP requests are handled (ex. handling of static file requests). The Web root of an ASP.NET Core application is configured using the "webroot" property in your project.json file.
+The web root of your app is the root location in your project from which HTTP requests are handled (for example handling of static file requests). The web root of an ASP.NET Core app is configured using the ``webroot`` property in the *project.json* file. When using the static files middleware only files from the web root folder are accessible; the other files in the content root **cannot** be accessed remotely. The default web root path is `<content root>/wwwroot`, but you can specify a different location using the `WebHostBuilder`.
```

Comment:
> The web root of an ASP.NET Core app is configured using the ``webroot`` property in the *project.json* file. 

Remove this sentence. It is no longer true. The web root is now configured using the WebHostBuilder (as you already say at the end of this paragraph).